retrofit/src/main/java/retrofit/RequestFactoryParser.java

```diff
@@ -211,7 +209,38 @@ private void parseParameters(Converter.Factory converterFactory) {
             throw parameterError(i, "Multiple Retrofit annotations found, only one allowed.");
           }
 
-          if (methodParameterAnnotation instanceof Path) {
+          if (methodParameterAnnotation instanceof Url) {
+            if (gotUrl) {
+              throw parameterError(i, "Multiple @Url method annotations found.");
+            }
+            if (gotPath) {
```

Comment:
Is there an order-of-parameters problem here? Ie. if I have `@Path` and then `@Url`, or the opposite?